test/code_analysis/compliant.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+
+# Author: Otavio Pontes
+# Email: otavio.pontes@intel.com
+
+check_sort_makefile()
+{
+	start_str="$1"
+	export LC_COLLATE=C
+	START=$(\grep "$start_str" Makefile.am --line-number -m 1|  cut -f1 -d:)
+	START=$((START + 1))
+	END=$(tail +"$START"  Makefile.am | \grep -e '^$' --line-number -m 1 |  cut -f1 -d:)
+	END=$((END - 1))
+	tail -n +$START Makefile.am | head -n $END | sort -c
+}
+
+@test "ANL003: Compliant code style check" {
+	run git diff --quiet --exit-code src
+	# shellcheck disable=SC2154
+	# output and status variable is being assigned and exported by bats
+	if [ "$status" -ne 0 ]; then
+		echo "Error: can only check code style when src/ are clean."
+		echo "Stash or commit your changes and try again."
+		return "$status"
+	fi;
+
+	run clang-format -i -style=file src/*.[ch] src/lib/*.[ch]
+	if [ "$status" -ne 0 ]; then
+		return "$status"
+	fi;
+
+	run git diff --quiet --exit-code src
+	if [ "$status" -ne 0 ]; then \
+		echo "Code style issues found. Run 'git diff' to view issues."; \
```

Comment:
same with the ; and \ in this section, no longer needed

Reply:
fixed